apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -81,9 +81,25 @@ public enum Reply
     /// </summary>
     public static TimeSpan MinimumAckExtensionWindow { get; } = TimeSpan.FromMilliseconds(50);
 
+    /// <summary>
+    /// The default minimum message Acknowledgement deadline extension of 0 seconds.
```

Comment:
0 seems unlikely to be useful. We should consult with the PubSub team about this.